StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -5,6 +5,10 @@
 #include "Components/RangeComponent.h"
 #include "EnemyFactory.h"
 #include "GameNetwork.h"
+#include "Player.h"
+// upgrades
+#include "Components/UpgradeComponents/UpgradeComponent.h"
```

Comment:
Do these need to be included in main, or can they be included in GameComponents.h instead?

Reply:
This is the first time I hear about GameComponents.h, and by the looks of it it might be unknown to others aswell. But it makes sense and I moved over component includes from main.